docs/MigrationGuide.md

```diff
@@ -13,6 +13,8 @@ To see the biggest differences please consult the [changelog](Changelog.md).
   Please update you Flink/Telegraf setup accordingly (see [nussknacker-quickstart](https://github.com/TouK/nussknacker-quickstart/tree/main/telegraf) for details). 
   If you still want to use `process` tag (e.g. you have a lot of dashboards), please set `countsSettings.metricsConfig.scenarioTag` setting to `process`
   Also, dashboard links format changed, see [documentation](https://docs.nussknacker.io/documentation/docs/installation_configuration_guide/DesignerConfiguration#metric-dashboard) for the details.
+* [#2645](https://github.com/TouK/nussknacker/pull/2645) Some of the default models (in particular `genericModel.jar`) 
```

Comment:
Can be precisely written which of them? Because "some of" can be mysterious a bit for our admins ;) Also what about `flinkExecutor.jar`?

Reply:
:+1: